habootstrap-formula.spec

```diff
@@ -34,6 +32,10 @@ BuildArch:      noarch
 Requires:       salt-shaptools
 Requires:       salt-formulas-configuration
 
+%define fname cluster
+%define fdir  %{_datadir}/salt-formulas
```

Comment:
whats the value for `%{_datadir}`

Reply:
This is a constant, it depends on the distro, in our case `/usr/share`